db/schema.rb

```diff
@@ -111,20 +111,22 @@
   create_table "pg_search_documents", force: :cascade do |t|
     t.text     "content"
     t.integer  "searchable_id"
-    t.string   "searchable_type", limit: 255
+    t.string   "searchable_type"
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
   create_table "users", force: :cascade do |t|
-    t.string   "first_name",      limit: 255
-    t.string   "last_name",       limit: 255
-    t.string   "email",           limit: 255
-    t.string   "password_digest", limit: 255
+    t.string   "first_name"
+    t.string   "last_name"
+    t.string   "email"
+    t.string   "password_digest"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "organization",    limit: 255
-    t.integer  "role",                        default: 2
+    t.string   "organization"
+    t.integer  "role",              default: 2
```

Comment:
I think we should make the default role 0 and change the `User::role` so the most basic role is first in the list.